tests/functional/disaster-recovery/regional-dr/test_disable_dr.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from time import sleep
+
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import tier1
+from ocs_ci.framework.pytest_customization.marks import turquoise_squad
+from ocs_ci.helpers import dr_helpers
+from ocs_ci.ocs import constants
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@turquoise_squad
+class TestDisableDR:
+    """
+    Test Disable Disaster Recovery
+
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["workload_type"],
+        argvalues=[
+            pytest.param(
+                *[constants.SUBSCRIPTION],
+                marks=pytest.mark.polarion_id("OCS-6209"),
+            ),
+            pytest.param(
+                *[constants.APPLICATION_SET],
+                marks=pytest.mark.polarion_id("OCS-6209"),
```

Comment:
Are polarion IDs same for this test? If yes, we can keep it constant and then use it.